data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java

```diff
@@ -862,12 +958,24 @@ public Date getDate(String columnName, Calendar calendar) throws SQLException {
 
     @Override
     public Time getTime(int i, Calendar calendar) throws SQLException {
-        return new Time(getDate(i, calendar).getTime());
+        return getTime(i);
```

Comment:
Why not involve the calendar for timezone info?

Reply:
Time is only hour, minute, second